### DISCUSSION TITLE
Replace current versionning system with versioneer

### DISCUSSION THREAD

**Discussion by skoudoro** [dipy/dipy#2302] [3. Community Lounge]
Does anyone mind if we replace the current versioning system with [versioneer](https://github.com/python-versioneer/python-versioneer)

it will make it easier to follow our weekly build, release management, etc...

Thanks for the feedback.

**Comment by arokem** [comment]
:+1:

If you haven't already, you might also consider [setuptools_scm](https://pypi.org/project/setuptools-scm/)

**Comment by skoudoro** [comment]
I did not start yet.  

I did not know `setuptools_scm`. What are the Pros/cons vs `versioneer`?

I need to try it.

**Comment by arokem** [comment]
The main advantage is that you don't need to vendor it into our code-base, but you instead add it as a dependency. 

It can be a bit wonky in some specific situations, but I have some experience hacking around the wonkiness.
